src/main.ts

```diff
@@ -859,25 +863,20 @@ export async function loadConfig(): Promise<Config> {
     }
   }
   core.info(`Resources will be resolved from ${shaArchive}`)
-
```

Comment:
Could you leave the lines blank? I think it makes the reading easier

Reply:
If we fold everything then there is not much need for blank lines because there is not much to read :)

I think it makes sense to fold initialization too.

Reply:
Maybe an empty before the result. I agree that can be helpful too.